### DISCUSSION TITLE
New Ways to Improve Civilian Reputation

### DISCUSSION THREAD

**Discussion by TG-Web-Team** [KillahPotatoes/KP-Liberation#934] [Ideas]
Would be cool to have tools for that. 

Such as using our ressources to "rebuild civilians infrastructures" after we've liberated a zone.

or use our ressources "provide humanitarian help"

These could be actual secondary missions or it could simply reduces our ressources (Supplies and fuel) which would  improve our reputation with civs

**Comment by UselessFodder** [comment]
Hey TG,

These are some good ideas. I'll look into seeing how difficult it would be to use FOB resources to rebuild buildings. I suspect not too bad, so I'll get back to you on that.

Of note, I did create a Secondary Objective mission that allows you to bring supplies to a friendly town/city and it will raise the civilian reputation if successful. Might be the sort of thing you're looking for. You can find it here: https://github.com/KillahPotatoes/KP-Liberation/compare/master...UselessFodder:KP-Liberation:RescueCivsMission
